bop_toolkit_lib/tests/README.md

```diff
@@ -0,0 +1,9 @@
+Simple tests for using multiprocessing with the BOP toolkit.
```

Comment:
I would change this line to "# Unit tests of the BOP toolkit" as the test below doesn't test just multiprocessing and we may want to add more tests in the future. I would then add a subsection "## Test of eval_bop19_pose.py" above the below command and table.